epoxy-adapter/src/main/java/com/airbnb/epoxy/ChangeAnimationDelegator.java

```diff
@@ -0,0 +1,168 @@
+package com.airbnb.epoxy;
+
+import android.animation.AnimatorListenerAdapter;
+import android.animation.EpoxyItemAnimation.AnimatorListener;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView.ItemAnimator;
+import android.support.v7.widget.RecyclerView.ViewHolder;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * ItemAnimator that wraps any other ItemAnimator and delegates change animations to
+ * {@link EpoxyModel}
+ */
+public final class ChangeAnimationDelegator extends ItemAnimatorDecorator
```

Comment:
Presumably if somebody is using `ChangeAnimationDelegator` they are using it with an EpoxyController so they know what Epoxy is, but I see your point.

The main reason I like the namespacing is that if you have a large app you might have another class called `ChangeAnimationDelegator` beside's Epoxy's. Then it is confusing which one is which. I see this a lot in our codebase. Small apps wouldn't really have that problem, but I tend to build for our use case and the problems I see on our team.